frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -632,6 +632,77 @@ mod tests {
 		})
 	}
 
+	#[test]
```

Comment:
Not a big fan of two separate tests for this. 100% and 0% are the two extremes and a bit hard to mentally follow. I suggest reading `unsigned_per_dispatch_checks_can_only_submit_threshold_better`, and creating a similar one.